token-swap/program/src/processor.rs

```diff
@@ -992,6 +1048,39 @@ impl Processor {
         Ok(())
     }
 
+    /// Processes a [SetFreezeAuthorityBitMask](enum.Instruction.html).
+    pub fn process_set_freeze_authority_bit_mask(
+        _: &Pubkey,
+        freeze_authority_bit_mask: u8,
```

Comment:
Setting a raw bitmask is fine, but be sure there's an easy library to do it from the outside!